### PR TITLE
feat: add new onboarding feature flag

### PR DESCRIPTION
**Pull request by MounirDhahri**
### Description

- Add new onboarding feature flag

### PR Checklist (tick all before merging)

- [x] I have not stored any sensitive keys/values/information in `Echo.json5` and in CI, or I have not changed anything in that file.

### CODE REVIEW

File: Echo.json5 [hunk 1]
```diff
@@ -37,6 +37,7 @@
     { name: 'AREnableOnlyTargetSupplyConsignments', value: true },
     { name: 'AREnableAuctionResultsKeywordFilter', value: true },
     { name: 'ARHomeAuctionResultsByFollowedArtists', value: true },
+    { name: 'AREnableNewOnboardingFlow', value: true },
```
Comment (pvinis) [inline Echo.json5:40]: 😬 `true`? did we have this as launch blocking? maybe we should start with `false`?

Comment (MounirDhahri) [reply Echo.json5:40]: good point, maybe better then to make this a draft until we decide later.

Comment (pvinis) [reply Echo.json5:40]: we could still add it, but with false for now. later, when we are ok to release, we make it true. echo is like a 2min change from PR to deploy anyway.

Comment (MounirDhahri) [reply Echo.json5:40]: sorry just saw the comment now, that makes sense but I think it's fine now since we didn't release it anyway :)

**Comment by MounirDhahri** [comment]
Time to go 🎉

**Pull request merged**